src/Documentation/fpga/xrt.rst

```diff
@@ -596,6 +596,72 @@ The driver object model looks like the following::
+      +-----+-----+             +-----+---+          +-----+----+
+      |           |             |         |          |          |
+      v           v             v         v          v          v
+ +------------+  +------+    +------+ +------+    +-----+ +-----------+
+ | xmgmt_main |  | VSEC |    | GPIO | | QSPI |    | CMC | | AXI-GATE0 |
+ +------------+  +------+    +------+ +------+    +-----+ +-----------+
+                             +---------+          +------+ +-------+
+                             | MAILBOX |          | ICAP | | CALIB |
+                             +---------+          +------+ +-------+
+                                                  +-----------+
+                                                  | AXI-GATE1 |
+                                                  +-----------+
+
+After an xclbin is download, group3 will be added and we have::
+
+                                +-----------+
+                                |   xmgmt   |
+                                +-----+-----+
+                                      |
+            +-------------+-----------+--------------------+
+            |             |           |                    |
+            v             |           v                    v
+       +--------+         |      +--------+            +--------+
+       | group0 |         |      | group1 |            | group2 |
+       +----+---+         |      +----+---+            +---+----+
+            |             |           |                    |
+            |             |           |                    |
+      +-----+-----+       |     +-----+---+          +-----+----+
+      |           |       |     |         |          |          |
+      v           v       |     v         v          v          v
+ +------------+  +------+ | +------+ +------+    +-----+ +-----------+
+ | xmgmt_main |  | VSEC | | | GPIO | | QSPI |    | CMC | | AXI-GATE0 |
+ +------------+  +------+ | +------+ +------+    +-----+ +-----------+
+                          | +---------+          +------+ +-------+
+                          | | MAILBOX |          | ICAP | | CALIB |
+                          | +---------+          +------+ +-------+
+                          v                      +-----------+
+                      +--------+                 | AXI-GATE1 |
+                      | group3 |                 +-----------+
+                      +---+----+
+                          |
+                          |
+     +----------------+---+-----------+--------------+
+     |                |               |              |
+     v                v               v              v
+ +--------+      +--------+      +--------+       +-----+
+ | CLOCK0 |      | CLOCK1 |      | CLOCK2 |       | UCS |
+ +--------+      +--------+      +--------+       +-----+
+ +-------------+ +-------------+ +-------------+
+ | CLOCK-FREQ0 | | CLOCK-FREQ1 | | CLOCK-FREQ2 |
+ +-------------+ +-------------+ +-------------+
+
```

Comment:
Same as before. But we should move group3 after group2 so it looks chronological?

Reply:
fixed